frontend/src/pages/MatchHistory.module.css

```
@@ -0,0 +1,5 @@
+.title {
+    text-align: center;
+    margin-bottom: 1rem;
+    color: var(--color-dark-green);
```

```suggestion
	margin: 1rem;

	font-family: 'Inter';
	font-style: normal;
	font-weight: 800;
	font-size: 32px;
	line-height: 32px;
	text-align: center;
	color: var(--color-white);
```
Keep the same style for all titles?

---

yeah that's okay, was unsure about the colors